src/main/scala/fairysf/variant/BreakthroughTroyka.scala

```diff
@@ -0,0 +1,40 @@
+package strategygames.fairysf
+package variant
+
+import strategygames.GameFamily
+
+case object BreakthroughTroyka
+    extends Variant(
+      id = 9, // unique within module fairySF
+      key = "breakthroughtroyka",
+      name = "Breakthrough",
+      standardInitialPosition = true,
+      fairysfName = FairySFName("breakthrough"),
+      boardSize = Board.Dim8x8
+    ) {
+
+  def gameFamily: GameFamily = GameFamily.BreakthroughTroyka()
+
+  def perfIcon: Char                      = 'a' // @TODO: adapt
+  def perfId: Int                         = 208 // just need to be unique accross all variants
+  override def canOfferDraw               = false
+  override def baseVariant: Boolean       = true
+  override def p1IsBetterVariant: Boolean = true
+
+  override def initialFen =
+    format.FEN("pppppppp/pppppppp/8/8/8/8/PPPPPPPP/PPPPPPPP w - - 0 1")
+
+  override def checkmate(situation: Situation) = false
+
+  override def staleMate(situation: Situation) = false
+
+  // The game ends with a loss if you run out of piece or if opponent reaches opposite final rank
+  override def specialEnd(situation: Situation): Boolean =
+    (situation.board.pieces.filter(e => e._2.player.p2).isEmpty
```

Comment:
Might be worth adding a `piecesOf` function to the `fairysf.Board` class to handle this nicely. You should be able to copy from `chess.Board` or `backgammon.Board`. it would then make this line:
`situation.board.piecesOf(Player.P1).isEmpty` which i think reads nicer.